Assets/Features/Managers/GameManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+
+namespace Features.Managers
+{
+    public class GameManager : MonoBehaviour
+    {
+        public static GameManager Instance;
+        
+        public AudioManager.Scripts.AudioManager AudioManager;
+        public EventManager EventManager;
```

Comment:
Properties might be better since we can now create a new instance and reassign it

Reply:
Added properties again with them seperated, becuase of the inspector issue